app/views/apply/eligibility/new-claim/same-journey-as-last-claim.html

```diff
@@ -16,22 +16,22 @@ <h1 class="govuk-heading-l">
       Your last journey
     </h1>
 
-    <table>
+    <table class="last-journey govuk-table">
```

Comment:
Is there a reason this isn't using the nunjucks macro? Not looked at what it actually does, just wondering 🙂